src/Joomlatools/Console/Joomla/Deserializer.php

```diff
@@ -0,0 +1,91 @@
+<?php
+	/**
+ * Copyright (C) Apis Networks, Inc - All Rights Reserved.
```

Comment:
We can't accept this file since this file is licensed to and owned by you. It might bring up discussions in the future. Since we also decided to only support YAML files this is not really an issue anymore, as we removed it in more recent comments.

For future changes like this, I would suggest you create a separate Composer package containing this class. It can then be included as another dependency in composer.json, preventing this issue in merges easily.

Reply:
That's my default licensing template. Go ahead and redact it.